File: Java/ArraysBidimensionals/src/ex01.java
public class ex01 {

	public static void main (String[] args){

		int[][] matrix = new int[5][5];

		for (int i = 0; i < matrix.length; i++) {

			for (int j = 0; j < matrix[i].length; j++) {

				matrix[i][j]=5;
			} 
		} 
     
		int i = 0;
		int j = 0;

		while (i < matrix.length) {
			j = 0;

			while(j < matrix[i].length){

				System.out.print (matrix[i][j]);
				j++;
			}
			System.out.println ();
			i++;
		}

	}

}

File: Base de dades/Php/recuExamen/ex01.php
<!DOCTYPE html>
<!--
To change this license header, choose License Headers in Project Properties.
To change this template file, choose Tools | Templates
and open the template in the editor.
-->
<html>
    <head>
        <meta charset="UTF-8">
        <title>Ex01</title>
    </head>
    <body>
        <?php
        // Ex01
        for ($i = 0; $i < 50; $i++) {
            $array[$i] = rand(0, 50);
            echo $i . "  ";
        } for ($i = 0; $i < 50; $i++) {
            if ($i == 0) {
                echo "<br>";
            }
            $array[$i] = rand(0, 50);
            echo $array[$i] . " ";
        } for ($i = 0; $i < 50; $i++) {
            if ($i == $array[$i]) {
                echo "<br> El número $array[$i] está en la posición $i.";
            }
        }
        ?>
    </body>
</html>
